docs/migrating_to_nextcord.rst

```diff
@@ -8,3 +8,43 @@ Migrating to nextcord
 Due to the `original discord.py repository <https://github.com/Rapptz/discord.py>`_ becoming read-only, we decided
 that it would be necessary to fork it and keep on developing further. We also wanted to change the name and voted on
 nextcord in order to properly register it at pypi.
+
+Porting from discord.py
+-------------------------
+
+In order to port a bot using discord.py to nextcord, follow these steps:
+
+1. Install nextcord: 
+
+.. code:: sh
+
+    # Linux/macOS
+    python3 -m pip install -U nextcord
+
+    # Windows
+    py -3 -m pip install -U nextcord
+
+2. Uninstall discord.py:
+
+..code:: sh
+
+    # Linux/macOS
+    python3 -m pip uninstall discord
+
+    # Windows
+    py -3 -m pip uninstall discord

```

Comment:
AFAIK it's `discord.py` not `discord`

Reply:
Hmm, discord works as an alias but I will change it because it is the recommendation on the discord pypi page. Thanks for the heads up.